examples/apps/quizmaster_cozmo.py

```diff
@@ -0,0 +1,403 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2017 Anki, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the file LICENSE.txt or at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Quiz-master Cozmo - a simple quiz game hosted by Cozmo.
+
+Uses the 3 cubes as buzzers for a quiz with up to 3 players.
+'''
+
+import asyncio
+import json
+from random import randrange, shuffle
+
+import cozmo
+
+
+class QuizQuestion:
+    """A single multiple choice question with 4 choices, one correct.
+    
+    Args:
+        question (str): The question.
+        options (list of str): 4 multiple choice answers where the
+            1st element is the correct answer. (They will be shuffled each time)
+    Raises:
+        :class:`ValueError` if not supplied exactly 4 options.
+    """
+    def __init__(self, question, options):
+        if len(options) != 4:
+            raise ValueError("Expected 4 options, got %s" % len(options))
+        self.question = question
+        self._answer_index = 0
+        self.options = list(options)  # copy the options, so we can shuffle them
+        self.shuffle_options()
+
+    @property
+    def answer_number(self):
+        """int: The number (1..4) representing the correct answer."""
```

Comment:
To describe the list in a way all people should understand, could do: (i.e., 1, 2, 3 or 4). I know in the SDK we already describe lists like (1..4), but for a higher level user who might want to use this example, instead I think we should use a more typical description.